spyder/widgets/status.py

```diff
@@ -57,7 +57,7 @@ def __init__(self, parent, statusbar, icon=None):
         self.set_icon(icon)
 
         # See spyder-ide/spyder#9044.
-        self.text_font = QFont(get_font(option='font'))
+        self.text_font = QFont(QFont().defaultFamily())
```

Comment:
@dalthviz why was this necessary?

Reply:
I asked @dalthviz to use the default system for our status bar instead of the monospaced one used in the Editor.

Reply:
Ahhh, ok :-) Got it!